examples/ffva/bsp_config/XK_VOICE_L71/platform/platform_conf.h

```diff
@@ -145,15 +145,15 @@
 /*  I/O Task Priorities                  */
 /*****************************************/
 #ifndef appconfQSPI_FLASH_TASK_PRIORITY
-#define appconfQSPI_FLASH_TASK_PRIORITY		    ( configMAX_PRIORITIES - 1 )
+#define appconfQSPI_FLASH_TASK_PRIORITY		    (configMAX_PRIORITIES-1)
 #endif /* appconfQSPI_FLASH_TASK_PRIORITY */
 
 #ifndef appconfI2C_TASK_PRIORITY
 #define appconfI2C_TASK_PRIORITY                (configMAX_PRIORITIES/2)
 #endif /* appconfI2C_TASK_PRIORITY */
 
 #ifndef appconfSPI_TASK_PRIORITY
-#define appconfSPI_TASK_PRIORITY                (configMAX_PRIORITIES/2)
+#define appconfSPI_TASK_PRIORITY                ( configMAX_PRIORITIES/2)
```

Comment:
Slightly odd extra space.

Reply:
Fixed in https://github.com/xmos/sln_voice/commit/795ac29743ceb59813f06646c5eefeda3d4ed274.